TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -199,6 +204,58 @@ bool TTSConfiguration::setSATPluginCallsign(const std::string callsign) {
     return false;
 }
 
+bool TTSConfiguration::setAccessList(const string &key,const string &value)
+{
+    std::map<std::string,std::string>::iterator itr;
+    itr = m_AccessList.find(key);
+    m_AclCalled = true;
+    if (itr != m_AccessList.end())
+    {
+        TTSLOG_INFO("method %s found in accesslist...replacing value with %s\n",key.c_str(),value.c_str());
+        itr->second= value;
+        return 1;
+    }
+    else
+    {
+        TTSLOG_INFO("method %s not found..inserting to accesslist with value %s\n",key.c_str(),value.c_str());
+        m_AccessList.insert(std::make_pair(key,value));
+        return 0;
+    }
+}
+
+bool TTSConfiguration::checkAccess(const string &method,string &callsign)
+{
+    if(m_AclCalled == true)
+    {
+        std::map<std::string,std::string>::iterator itr;
+        itr = m_AccessList.find(method);
+        if (itr != m_AccessList.end())
+        {
+            string value = itr->second;
+            string app_quote =  '\"' + callsign + '\"';//wrap it with double quote
+            std::string::size_type pos = value.find(app_quote);
```

Comment:
there is no need to store the result of find()

Reply:
ok